SSSwiftUIAnimations/Sources/ReactionAnimation/ReactionAnimationViewModel.swift

```diff
@@ -0,0 +1,193 @@
+//
+//  AnimationViewModel.swift
+//  ReactionAnimationSwiftUI
+//
+//  Created by Faaiz Daglawala on 15/12/22.
+//
+
+import Foundation
+//import Observation
+import SwiftUI
+
+/// ViewModel for managing the reaction animation in SwiftUI.
+class SSReactionAnimationViewModel: ObservableObject {
```

Comment:
We can mark this as public if we are using it outside the package.

Reply:
We are now not using this outside the package now, as there is a new style struct created which is used outside the package.